playbackSpeed/src/main/kotlin/voice/playbackSpeed/PlaybackSpeedController.kt

```diff
@@ -0,0 +1,23 @@
+package voice.playbackSpeed
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun PlaybackSpeedController(speed: Float, onValueChange: (Float) -> Unit) {
```

Comment:
I can't quite make sense of the naming. What is the reason for a `PlaybackSpeedController` and `PlaybackSpeedComponent`.
Also they have lots of default values that I recommend to inline.
I would only make things configurable that are actually configured.
Else it is quite hard to follow the logic flow.